README.md

```diff
@@ -1,8 +1,11 @@
+<div style="width:75%; margin: auto; margin-top: 50px; margin-bottom: 50px;">
+
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="./docs/assets/logo/qadence_logo_white.svg">
-  <source media="(prefers-color-scheme: light)" srcset="./docs/assets/logo/qadence_logo.svg">
-  <img alt="Qadence logo" src="./docs/assets/logo/qadence_logo.svg">
+  <source media="(prefers-color-scheme: dark)" srcset="./assets/logo/qadence_logo_white.svg">
+  <source media="(prefers-color-scheme: light)" srcset="./assets/logo/qadence_logo.svg">
+  <img alt="Qadence logo" src="./assets/logo/qadence_logo.svg">
 </picture>
+</div>
```

Comment:
Ok apparently this broke the github readme

Reply:
Ah wait

Reply:
?